Options.py

```diff
@@ -785,6 +785,7 @@ class VerifyKeys(metaclass=FreezeValidKeys):
     verify_item_name: bool = False
     verify_location_name: bool = False
     value: typing.Any
+    display_valid_keys: bool = False  # Should the full list of keys appear in templates
```

Comment:
```suggestion
    display_valid_keys: bool = False
    """ Should the full list of keys appear in templates """
```
I understand if you don't want to do this because it takes up more space.
Putting the doc string here enables the IDE to show that information in mouseover tooltip.
![member-doc-strings](https://github.com/ArchipelagoMW/Archipelago/assets/6052147/d29e3a34-e682-4fff-a447-ac8cc62a0e20)